apps/dsp-app/src/app/project/list/list.component.ts

```diff
@@ -123,22 +115,20 @@ export class ListComponent extends ProjectBase implements OnInit, OnDestroy {
 
   askToDeleteList(list: ListNodeInfo): void {
     this._dialog
-      .afterConfirmation('Do you want to delete this controlled vocabulary?', list.labels[0].value)
+      .afterConfirmation('Do yu want to delete this controlled vocabulary?', list.labels[0].value)
+      .pipe(
+        take(1),
+        switchMap(() => this.listIri$.pipe(map(listIri => this._store.dispatch(new DeleteListNodeAction(listIri)))))
+      )
+      .pipe(switchMap(() => this._actions$.pipe(ofActionSuccessful(DeleteListNodeAction), take(1))))
```

Comment:
please stop using ofActionSuccessful all the time, as this.store.dispatch(new DeleteListNodeAction) returns it

Here
``
.pipe(
        take(1),
        switchMap(() => this.listIri$),
        switchMap(listIri =>  this._store.dispatch(new DeleteListNodeAction(listIri))
      )
``